CafeTime/CafeTime/CafeRealm.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CafeRealm.swift
+//  CafeTime
+//
+//  Created by Vladysalv Vyshnevksyy on 2/7/17.
+//  Copyright © 2017 Vladysalv Vyshnevksyy. All rights reserved.
+//
+
+import UIKit
+import RealmSwift
+
+class CafeRealm: Object {
```

Comment:
why is this class called CafeRealm? this is just a simple model of Cafe instance that has nothing to do with Realm